ynr/apps/parties/importer.py

```diff
@@ -363,3 +363,45 @@ def get_default(self):
             DEFAULT_EMBLEMS.get(self.party.ec_id, False)
             == self.emblem_dict["Id"]
         )
+
+
```

Comment:
So are we comparing parsed SOPN descriptions to the data from the YNR live site instead of or in addition to the live EC data? And how often does YNR or EE import party registration information from EC?

Reply:
We are comparing the parsed descriptions against whatever we have stored in our local DB, so it does depend on when we have last run the `make populate-sopn-testing-database` last. But this change means that it will now import parties from the EC, and then also any other `PartyDescription` objects that are stored in YNR but are no longer kept track of by the EC. So when this is merged its worth running the importer so that you have the extra `PartyDescription` objects stored in your local db.

YNR runs the import from the EC every night around 2am.